Dummy_disease_test/data/index.json

```diff
@@ -1,5 +1,4 @@
 {
-    "version": 2,
```

Comment:
Was no schema here intentional?

Reply:
Yep. I've made two changes: one to add the `$schema` property and another to drop the (now-redundant) `version` property. Is that ok?

Reply:
Changed my mind :upside_down_face:. I'll leave the `version` property there for now to avoid breaking backwards compatibility. We can remove it properly when we make a new config file format.

Reply:
Cool :+1: